.pre-commit-config.yaml

```diff
@@ -5,6 +5,7 @@ repos:
     -   id: black
         args: [--safe, --quiet]
         language_version: python3
+        additional_dependencies: [click==8.0.2]
```

Comment:
It doesn't need to be in conda env file for this additional dependencies!?

Reply:
no. It is installed by pre-commit on an exclusive env.